python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -513,7 +513,14 @@ def fillout_available_node_types_resources(
             if instance_type in instances_dict:
                 cpus = instances_dict[instance_type]["VCpuInfo"][
                     "DefaultVCpus"]
-                autodetected_resources = {"CPU": cpus}
+                memory_total = instances_dict[instance_type]["MemoryInfo"][
+                    "SizeInMiB"]
+                memory_total = int(memory_total) * 1024 * 1024
+                memory_resources = int(memory_total * 0.6)
```

Comment:
should this be `memory_total * (1 - ray_constants.DEFAULT_OBJECT_STORE_MEMORY_PROPORTION)`?

Reply:
Hi @wuisawesome, I referred here https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/example-full.yaml#L133